templates/install/project.md

```diff
@@ -104,22 +104,33 @@ Also, you can get the Lean documentation inside VS Code using
 inside the text field that appears, type "lean doc" and hit <kbd>Enter</kbd>.
 Then click "Mathematics in Lean" or "Theorem Proving in Lean" and enjoy.
 
-## Hosting your project on GitHub
+## Updating Mathlib in your project
 
-Your project is already a git repository, and as it grows,
-you may want to host it on [GitHub](https://guides.github.com/activities/hello-world/).
-If you take this step, the community offers some
-[GitHub Actions scripts](../ci.html) that could help manage your repository.
-But don't worry if you don't know what this means.
-It's not necessary for using Lean.
+If you have a project depending on Mathlib, and you want to update to the latest version of Lean and Mathlib, you have to do two steps:
+* Update the `lean-toolchain` file in your repository to the latest version. You can do this automatically by running the following in the root directory of your repository.
+```
+curl -L https://raw.githubusercontent.com/leanprover-community/mathlib4/master/lean-toolchain -o lean-toolchain
+```
+* run `lake -R -Kenv=dev update`. This will update Lean, Mathlib and download the new Mathlib cache for you.
+
+You should then check whether all files in your repository still compile.
+The most convenient way to do this is to run `lake exe mk_all`. However, that might not work on Windows at the moment.
```

Comment:
As I understand it, mk_all works on Windows now - so we can remove the `find` incantation, right?